src/api/status/public/js/posts-count.js

```diff
@@ -0,0 +1,18 @@
+const totalPostsEl = document.getElementById('totalPosts');
+const fetchPostsCount = () =>
+  fetch('https://dev.api.telescope.cdot.systems/v1/posts').then((res) =>
```

Comment:
This should be using `process.env.API_HOST` instead. In production, `process.env.API_HOST` is  `dev.api.telescope.cdot.systems`

https://github.com/Seneca-CDOT/telescope/blob/master/config/env.production#L21

Reply:
Also, you `fetch(url, {method: 'HEAD'})`

https://github.com/Seneca-CDOT/telescope/pull/2471/files#r752232958

Reply:
> process.env.API_HOST

will it work? As I know, the browser cannot read environment variable.

Reply:
Correct, we can't read `process` in the browser.

What we need here is a function to get the API Host for us, something like:

```js
const apiHost = new URL(location).origin;

...

fetch(`${apiHost}/v1/posts`)
```